code/clue.js

```diff
@@ -4,56 +4,191 @@ const mrGreen = {
+}
+const profPlum = {
+  firstName: 'Victor',
+  lastName: 'Plum',
+  color: 'plum',
+  description: 'Disgraced former psychiatrist',
+  age: 39,
+  image: 'assets/plum.png',
+  occupation: 'Professor'
+}
+const missScarlet = {
+  firstName: 'Cassandra',
+  lastName: 'Scarlet',
+  color: 'red',
+  description: 'Young, cunning and attractive',
+  age: 25,
+  image: 'assets/scarlet.png',
+  occupation: 'Madame'
+}
+const mrsPeacock = {
+  firstName: 'Eleanor',
+  lastName: 'Peacock',
+  color: 'blue',
+  description: 'A grande dame',
+  age: 54,
+  image: 'assets/peacock.png',
+  occupation: 'Shady housevife'
+}
+const cMustard = {
+  firstName: 'Colonel',
+  lastName: 'Mustard',
+  color: 'yellow',
+  description: 'A dignified, dapper and dangerous military man',
+  age: 61,
+  image: 'assets/mustard.png',
+  occupation: 'Military officer'
+}
+const mrsWhite = {
+  firstName: 'Mrs',
+  lastName: 'White',
+  color: 'white',
+  description: 'He has a lot of connections',
+  age: 48,
+  image: 'assets/white.png',
+  occupation: 'Servant'
 }
 
 // CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
 
 const rope = {
-  name: 'rope',
-  weight: 10
+  name: 'Rope',
+  weight: 10,
+}
+const knife = {
+  name: 'Knife',
+  weight: 10,
+}
+const candlestick = {
+  name: 'Candlestick',
+  weight: 10,
+}
+
+const dumbbell = {
+  name: 'Dumbbell',
+  weight: 10,
+}
+const poison = {
+  name: 'Poison',
+  weight: 10,
+}
+
+const axe = {
+  name: 'Axe',
+  weight: 10,
+}
+const bat = {
+  name: 'Bat',
+  weight: 10,
+}
+const trophy = {
+  name: 'Trophy',
+  weight: 10,
+}
+const pistol = {
+  name: 'Pistol',
+  weight: 10,
+
+
 }
 
 // THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+//2 NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
 
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
+  mrsWhite,
+  profPlum,
+  missScarlet,
+  mrsPeacock,
+  cMustard
 ]
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol,
+]
 
-const rooms = []
+const rooms = [
+  'Dining Room',
+  'Conservatory',
+  'Kitchen',
+  'Study',
+  'Library',
+  'Biliard Room',
+  'Lounge',
+  'Ballroom',
+  'Hall',
+  'Spa',
+  'Living Room',
+  'Observatory',
+  'Theater',
+  'GuestHouse',
+  'Patio',
+]
 
 // THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
 // YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
-const randomSelector = array => {
-  return array[Math.floor(Math.random() * array.length)]
-}
-
+const randomSelector = arrays => {
+  return arrays[Math.floor(Math.random() * arrays.length)]
+} 
 // CREATE AN OBJECT THAT KEEPS THE MYSTERY.
 // With a killer, a weapon and a room.
 // The values will be set later.
+const mystery = {
+  killer: {},
+  weapon: {},
+  room: {},
+};
```

Comment:
seems like there was many ways of creating an object with a empty value, have seen the exampels also...

Reply:
It's even more semantic approach to initialise mystery object with null values like this :

```
const mystery = {
  killer: null,
  room: null,
  weapon: null
}
```